rfc6962-bis.xml

```diff
@@ -338,7 +338,7 @@ d0 d1   d2 d3           d0 d1   d2 d3  d4 d5</artwork>
     </section>
     <section title="Private Domain Name Labels">
       <t>
-        Some regard some DNS domain name labels within their registered domain space as private and security sensitive. Even though these domains are often only accessible within the domain owner's private network, it's common for them to be secured using publicly trusted TLS server certificates. We define a mechanism to allow these private labels to not appear in public logs.
+        Some organizations regard certain DNS domain name labels within their registered domain space as private and security sensitive. Even though these domains are often only accessible within the domain owner's private network, it's common for them to be secured using publicly trusted TLS server certificates. We define a mechanism to allow these private labels to not appear in public logs.
```

Comment:
In -04 of 6962-bis, we changed "Some enterprises regard..." to "Some regard...", because enterprises might not be the only parties that want redaction.

What if a domain name is registered by an individual rather than an organization?

How about: "Some domain registrants regard certain DNS domain name labels within their registered domain space as private and security sensitive." ?